tests/js/client/agency/agency-test.js

```diff
@@ -377,6 +377,140 @@ function agencyTestSuite () {
 
 ////////////////////////////////////////////////////////////////////////////////
 /// @brief Test transact interface
+////////////////////////////////////////////////////////////////////////////////
+
+    testPoll : function () {
+      var cur = accessAgency("write",[[{"/": {"op":"delete"}}]]).
+          bodyParsed.results[0];
+      var ret, result, log, ci, job, wr;
+
+      ret = request({url: agencyLeader + "/_api/agency/poll",
+                         method: "GET", followRedirect: true});
+      if (ret.statusCode === 202) {
```

Comment:
Do we answer with 202?

Reply:
See comment above, I would have thought 200 is more appropriate.

Reply:
Furthermore, let's assert on the status code. Otherwise, if it is wrong, we have a strange error message with an exception further down.

Reply:
202 just passed on the RestHandlers's code.  adjusted